Training24/TestCoinReturn.c

```diff
@@ -0,0 +1,73 @@
+// ------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// Dhaarani Mohanasundaram
+// -------------------------------------
+// TestCoinReturn.c
+// Program on Test2.2 branch.
+// ------------------------------------------------------------------------------------
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+#include "CoinReturn.h"
+
+#define RED "\x1b[31m"
+#define GREEN "\x1b[32m"
+#define RESET "\x1b[0m"
+
+void TestCoinReturn (int testNumber, int paidAmount, int actualAmount, int expectedCoins[], int* coins, int numCoins) {
+   int coinsReturned[5] = { 0 };
+   CalculateChange (paidAmount, actualAmount, coinsReturned, coins, numCoins);
+   if (coinsReturned[0] == -1) {
+      int isCoinsMatch = (expectedCoins[0] == -1);
+      printf ("Test %d: %s\nPaid amount: %d, Actual amount: %d; Insufficient amount paid.\n\n",
```

Comment:
```suggestion
      printf ("Test %d: %s\nPaid amount: %d, Actual amount: %d; \n\n",
```
Tests should not print unnecessary messages unless a test is failing and you need to indicate what/where.